### DISCUSSION TITLE
split_raster() function outputs are distorted

### DISCUSSION THREAD

**Discussion by MoAbbasid** [weecology/DeepForest#683] [Q&A]
The preprocess.split_raster() output crops are distorted.
Im running the same code on the training notebook, Im trying to test finetuning a model for more dense forest imagery to see how it performs.

code:
`crop_dir = "/content/train_data_folder"
train_annotations= preprocess.split_raster(path_to_raster=YELL_train,
                                 annotations_file=annotation_path,
                                 base_dir=crop_dir,
                                 patch_size=400,
                                 patch_overlap=0.05)`

I just changed the output dir
Screenshots:
![image](https://github.com/weecology/DeepForest/assets/109457597/d3129e01-f3e7-4b58-aade-b76ed07691cb)
![image](https://github.com/weecology/DeepForest/assets/109457597/fdaa376d-787c-4db3-ba7b-206728d16b62)

not sure if its related to the way the pngs are cropped or the way they are saved.
any idea why this happens?

Thanks

**Comment by bw4sz** [comment]
There is no error here. This is what the test data from NEON looks like. Completely unrelated to DeepForest. They georectify using a LIDAR mask that distorts the RGB imagery. Check out the original image, you will see it there as well. Closing.

**Reply by MoAbbasid** [reply]
@bw4sz, you're right, I looked at the large area and it does indeed look like that,
bare with me these noob questions while I familiarize myself with deepforest
